samples/nrf9160/lwm2m_client/overlay-nbiot.conf

```diff
@@ -1,4 +1,5 @@
-CONFIG_LTE_NETWORK_MODE_NBIOT=y
+CONFIG_LTE_NETWORK_MODE_NBIOT_GPS=y
+CONFIG_LTE_NETWORK_MODE_LTE_M_GPS=n
```

Comment:
nit: It's enough to set `CONFIG_LTE_NETWORK_MODE_NBIOT_GPS=y` - as these options are part of a single `choice` entry, they're exclusive - enabling one of them automatically invalidates the other.

Reply:
Good to know, thanks.